cardano-api/src/Cardano/Api/Fees.hs

```diff
@@ -1145,7 +1146,7 @@ makeTransactionBodyAutoBalance systemstart history pparams
     | txOutValueToLovelace balance < 0 =
         Left . TxBodyErrorAdaBalanceNegative $ txOutValueToLovelace balance
     | otherwise =
-        case checkMinUTxOValue (TxOut changeaddr balance TxOutDatumNone ReferenceScriptNone) bpparams of
+        case eBpparams >>= checkMinUTxOValue (TxOut changeaddr balance TxOutDatumNone ReferenceScriptNone) of
```

Comment:
Here, I would take `bpparams` as an argument.

Reply:
Do you mean accepting `Either` as argument? Can you elaborate what do you have in mind?